_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -4,17 +4,27 @@ order: 104
 ---
 #  {{ page.title }}
 
-Note: If you used the Forseti setup wizard to deploy, your `deploy-forseti.yaml` 
-will have a timestamp suffix, e.g. `deploy-forseti-20171001000000.yaml`.
+This page describes how to change or _upgrade_ an existing installation of Forseti.
+
+There are four steps steps to upgrading existing deployments of Forseti.
+
+ 1. [Upgrade the codebase](#get-the-latest-version)
+ 1. [Check for differences in the deployment](#check-for-changes-to-the-deployment-template)
+ 1. [Check for and enable any new capabilities](#enable-new-features)
+ 1. [Upgrade the deployment](#update-the-deployment)
+
+## Get the latest version
 
-### Get a particular deployment template version
-This step is **optional**.
+This step is **optional** if you're not changing versions of Forseti.
 
-If you need to change the `release-version` for your Forseti deployment, you MUST 
+If you need to change the `release-version` for your Forseti deployment, you **MUST** 
 get the correct version of Forseti. The deployment template's startup 
 script has release-specific code, so things will break if you use a startup script that 
 is out of sync with the deployed release.
 
+Note: If you used the Forseti setup wizard to deploy your `deploy-forseti.yaml` 
```

Comment:
Maybe a comma between "to deploy" and "yoour `deploy-forseti.yaml`"

Reply:
Improved, thanks.